src/Umbraco.Web.UI.Client/src/less/components/overlays.less

```diff
@@ -148,7 +148,14 @@
 
 @media (max-width: 767px) {
     .umb-overlay.umb-overlay-left {
-        margin-left: 60px;
+        margin-left: 61px;
+    }
+}
+
+@media (max-width: 500px) {
+    .umb-overlay.umb-overlay-left {
+        margin-left: 41px;
+        width: auto;
```

Comment:
Yes, I noticed that too a few times. I think it happened when clicking on avatar or help before the page was full rendered or on first click.

Not sure if width: auto !important makes a difference although it should be avoided.

But if calc works it is great. Before the 500px width was just too wide so the close link was hidden.